backend/api/firebase_auth.py

```
@@ -0,0 +1,64 @@
+            uid = decoded_token['uid']
+            email = decoded_token.get('email', '')
+            
+            # Get or create the user
+            try:
+                user = User.objects.get(username=uid)
+            except User.DoesNotExist:
+                # Create a user with the uid as username
+                user = User(
+                    username=uid,
+                    email=email,
+                    is_active=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)